packages/console/src/pages/SignInExperience/components/GuideModal.tsx

```diff
@@ -28,10 +29,12 @@ import TermsForm from './TermsForm';
 
 type Props = {
   isOpen: boolean;
+  languages: LanguageTag[];
+  onMutateLanguages: () => void;
```

Comment:
we can use `mutate(url)` to avoid the deep prop chain.
<img width="718" alt="image" src="https://user-images.githubusercontent.com/5717882/194693553-79dea124-21c7-486e-98d4-b65e6b2a1e18.png">

Reply:
cool